.travis.yml

```diff
@@ -47,6 +47,7 @@ script:
   - date --rfc-3339=seconds
   - nvm install
   - cp .env_sample .env
+  - source .env
```

Comment:
Is it OK to source the `.env` file? All I need is the `ELASTICSEARCH_URL` environment variable, but I see there are some globals vars set in Travis. I think the only ones it'd override are GitHub ones that already have fake values.

Reply:
I'm not sure this is damaging, it might override others vars set in Travis though giving a misleading expectation to the administrator that sets them there. Maybe this is not a problem since we define variables in this file as well.

What if you just add `ELASTICSEARCH_URL` to the `global` section up in the file?

Reply:
Yeah, I can definitely do that. It was my initial thought, but I had thought it made sense to use our `.env` even though it's copied from the sample `.env`. I'll make the change.